lib/AST/ASTContext.cpp

```diff
@@ -22,11 +22,18 @@ extractLibraries(const std::vector<std::string> &LibrariesAddressInfo) {
     std::string Buff;
     while (SS >> Buff) {
       std::string LibName, Address;
-      size_t I = 0;
-      for (; Buff.at(I) != ':'; ++I)
-        LibName += Buff.at(I);
-      for (++I; I < Buff.size(); ++I)
+      if (Buff.empty())
+        continue;
+      size_t I = Buff.size() - 1;
```

Comment:
Should we use `string::find` and `string::substr` to do this logic? I think it will be more clear and avoid calling `+=` for several time.

Reply:
fixed